buildHelpers/createRollupConfig.js

```diff
@@ -0,0 +1,103 @@
+import typescript from '@rollup/plugin-typescript';
+import { terser } from 'rollup-plugin-terser';
+
+// type RollupOptions = {
+//   format: 'umd' | 'cjs' | 'esm',
+//   isExplorer: boolean,
```

Comment:
Coould we maybe rename this to `isSandbox`? it feels like sandbox is kinda still explorer, but the other way isn't true so might be a bit clearer?